reana_workflow_controller/consumer.py

```diff
@@ -64,6 +65,7 @@ def get_consumers(self, Consumer, channel):
                 queues=self.queue,
                 callbacks=[self.on_message],
                 accept=[self.message_default_format],
+                prefetch_count=REANA_JOB_STATUS_CONSUMER_PREFETCH_COUNT,
```

Comment:
Could you perhaps do `prefetch_count=ENV if ENV else False`  so that we could use ENV=0 as a way to switch off the prefetch count?

(I tested 1 and 200 and there did not seem to have been much visible difference...)

Reply:
There will be no visible difference in terms of performance (value `200` vs unlimited) with a few workflows, even with a couple of hundreds, I think. Prefetch doesn't improve the performance of our code, jobs-status consumer will work as it was working before. It limits the buffer size so RAM consumption will not go crazy if we will launch 100k workflows and jobs-status consumer will suddenly prefetch 10k+ messages and eat RAM from the node.